### ISSUE TITLE
Different expected result from running test on Linux

### ISSUE THREAD

**Issue by char0n** [swagger-api/swagger-js#1511]
Q&A (please complete the following information)
 - OS: [e.g. Linux]
 - Environment: [Node.js v13.3.0]
 - Method of installation: [e.g. npm, unpkg]
 - Swagger-Client version: [commit: 2b950ee77f814069b9f1d92a422eeb56c47ac2b5]

### Content & configuration

`npm run test:unit`

### To reproduce...

`npm run test:unit`

One test if failing on:

` request to https://localhost:8000/petstore.json failed, reason: write EPROTO 140678765725568:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:`

### Expected behavior

`request to https://localhost:8000/petstore.json failed, reason: socket hang up`

### Additional context or thoughts

Probably depends on platform or openssl version.
Affected test: https://github.com/swagger-api/swagger-js/blob/2063f8e8cbdb35e0493e32014e40d561ccfdbe59/test/client.js#L284

**Comment by char0n** [comment]
Analysis:

**Expected results on:**
```
lts/boron -> v6.17.1
lts/carbon -> v8.17.0
```

**Unexpected results on:**
```
lts/dubnium -> v10.20.1
lts/erbium -> v12.16.3
v13.y.z
v14.y.z
```

Will provide a PR compensating for this.

**Comment by char0n** [comment]
Closed by #1522
